v0/pipeline/steps/cd/aws-cdk-deploy-step-node.yaml

```diff
@@ -0,0 +1,55 @@
+title: AwsCdkDeployStepNode
```

Comment:
move this file to common module steps/common as this is being referred by multiple modules

Reply:
Done